.github/workflows/publish-mac-arm.yml

```diff
@@ -0,0 +1,57 @@
+name: Publish MacOS-ARM release
+
+on:
+  workflow_dispatch:
+  
+permissions:
+  id-token: write 
+  contents: write
+
+jobs:
+  get_release_version:
+    name: Get Release Version
+    runs-on: ubuntu-latest
+    if: github.repository_owner == 'ballerina-platform'
+    steps:
+      - id: checkout
+        uses: actions/checkout@v2
+      - id: execute
+        run: |
+          export VERSION="$((grep -w 'version' | cut -d= -f2) < gradle.properties | cut -d- -f1)"
+          echo "::set-output name=version::${VERSION}"
+    outputs:
+      version: ${{steps.execute.outputs.version}}
```

Comment:
@Miranlfk Shall we include version as an user input?

Reply:
This has been completed